liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -39,6 +39,7 @@ public class ColumnSnapshotGenerator extends JdbcSnapshotGenerator {
     private static final Pattern POSTGRES_NUMBER_VALUE_PATTERN = Pattern.compile(POSTGRES_NUMBER_VALUE_REGEX);
 
     private static final String MYSQL_DEFAULT_GENERATED = "DEFAULT_GENERATED";
+    private static final String POSTGRES_GENERATED_ALWAYS = "GENERATED ALWAYS AS ";
```

Comment:
If I'm not wrong this is also supported also by Oracle. I think maybe we can rename this constant and use it for both cases. Have you tried this in Oracle as well?

Reply:
So, PostgreSQL and Oracle cases are close, but I see some differences.

1. Different syntax. For PostgreSQL, it will be `GENERATED ALWAYS AS (value) STORED`. So, we have this STORED at the end.
2. It Looks like Oracle returns us the default value without "()", but PostgreSQL returns with:

![2023-12-22 19_43_20-DBeaver  C__Users_anvolka_work_DbeaverP_dbeaver-workspace-idea  - C__Users_anvol](https://github.com/liquibase/liquibase/assets/45152336/e40c7db3-b953-4be9-8e55-35a3145af29e)

![2023-12-22 19_39_18-DBeaver  C__Users_anvolka_work_DbeaverP_dbeaver-workspace-idea  - C__Users_anvol](https://github.com/liquibase/liquibase/assets/45152336/e8ff1ba1-a718-4653-aba3-88c4af371d0e)

3. We have different conditions for column flags. 
It is `"YES".equals(columnMetadataResultSet.get("VIRTUAL_COLUMN")` for Oracle.
And `"YES".equals(columnMetadataResultSet.get("IS_GENERATEDCOLUMN"))` for PostgreSQL.

So, I can add all these conditions as flags in one merged method. But I am unsure.

Reply:
I think we shouldn't merge conditions, I would leave it as it's now.